tests/kafkatest/services/performance/producer_performance.py

```diff
@@ -38,12 +41,14 @@ def __init__(self, context, num_nodes, kafka, security_protocol, topic, num_reco
         }
         self.settings = settings
         self.intermediate_stats = intermediate_stats
+        self.client_id = client_id
 
     def _worker(self, idx, node):
         args = self.args.copy()
-        args.update({'bootstrap_servers': self.kafka.bootstrap_servers()})
-        cmd = "/opt/kafka/bin/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance "\
-              "%(topic)s %(num_records)d %(record_size)d %(throughput)d bootstrap.servers=%(bootstrap_servers)s" % args
+        args.update({'bootstrap_servers': self.kafka.bootstrap_servers(), 'jmx_port': self.jmx_port, 'client_id': self.client_id})
+        cmd = "JMX_PORT=%(jmx_port)d /opt/kafka/bin/kafka-run-class.sh org.apache.kafka.clients.tools.ProducerPerformance " \
+              "%(topic)s %(num_records)d %(record_size)d %(throughput)d bootstrap.servers=%(bootstrap_servers)s client.id=%(client_id)s" \
+              " | tee /mnt/producer-performance.log" % args
```

Comment:
@lindong28 
I tried this a bit locally, and realized the command now has two pipes to `tee` (see line 58 as well)

When I drop the pipe to tee here on line 51 and keep the one below, the producer runs as expected.